ipfs/filestore.go

```diff
@@ -183,12 +157,12 @@ func (a *Adder) Close() error {
 }
 
 func (fs *Filestore) NewAdder(pin, wrap bool) (cafs.Adder, error) {
-	node := fs.Node()
+	node := fs.node
 	ctx := context.Background()
-	bserv := blockservice.New(node.Blockstore, node.Exchange)
-	dagserv := dag.NewDAGService(bserv)
+	// bserv := blockservice.New(node.Blockstore, node.Exchange)
```

Comment:
Either delete these lines, or add a TODO about what needs to be down to revive them.